src/main/java/com/github/tomakehurst/wiremock/jetty9/JettyHttpServer.java

```diff
@@ -102,7 +107,13 @@ public JettyHttpServer(
 
         applyAdditionalServerConfiguration(jettyServer, options);
 
-        jettyServer.setHandler(createHandler(options, adminRequestHandler, stubRequestHandler));
+        final HandlerCollection handlers = createHandler(options, adminRequestHandler, stubRequestHandler);
+        jettyServer.setHandler(new HandlerCollection(ArrayUtils.insert(0, handlers.getHandlers(), new AbstractHandler() {
```

Comment:
How about adding this inside the HandlerCollection created in `createHandler`? Might be a bit neater than accepting a collection then re-packing it into another.